### PR TITLE
gh #14 l1 code cleanup

### PR DESCRIPTION
**Pull request by thanushreevani**
- Code cleanup and build.sh update
1. Changing the indentation tab space to spaces in whole file
2. in whole doc steps are not added inside API code..so added it
3. edited numbering in steps if order is not correct
- Re-checked if any new API interface has been added recently and requires an L1 test case, no new API interface was found.

### CODE REVIEW

File: src/test_l1_rmfAudioCapture.c [hunk 1]
```diff
@@ -100,75 +100,75 @@ bool test_l1_is_aux_capture_supported()
...
-	else
-		return false;
+    const char * config = getenv("AC_GENERATE_POSITIVE_ONLY_SUITE");
+    if((NULL != config) && (0 == strncasecmp(config, "TRUE", 4)))
+        return true;
+    else
+        return false;
 }
 
 rmf_Error test_l1_dummy_data_cb(void *cbBufferReadyParm, void *AudioCaptureBuffer, unsigned int AudioCaptureBufferSize)
```
Comment (Sumip98) [inline src/test_l1_rmfAudioCapture.c:110]: Please include the necessary test steps in the entire file

File: src/test_l1_rmfAudioCapture.c [hunk 2]
```diff
@@ -429,56 +455,66 @@ void test_l1_rmfAudioCapture_positive_RMF_AudioCapture_Close (void)
...
-* | 11 | Call `RMF_AudioCapture_Stop()` to stop capture | valid handle | RMF_SUCCESS | Should pass |
-* | 12 | Call `RMF_AudioCapture_Close()` to release resources, test open->start->stop->start->stop->close | current handle | RMF_SUCCESS | Should pass |
+* | 06 | Call `RMF_AudioCapture_Open()` to open interface | handle must be a valid pointer | RMF_SUCCESS | Should pass |
+* | 07 | Call `RMF_AudioCapture_Start()` to start audio capture | settings=default settings , but dummy data callback, status callback NULL | RMF_SUCCESS | Should pass |
+* | 08 | Call `RMF_AudioCapture_Stop()` to stop capture | valid handle | RMF_SUCCESS | Should pass |
+* | 09 | Call `RMF_AudioCapture_Start()` to start audio capture | settings=default settings , but dummy data callback, status callback NULL | RMF_SUCCESS | Should pass |
+* | 10 | Call `RMF_AudioCapture_Stop()` to stop capture | valid handle | RMF_SUCCESS | Should pass |
+* | 11 | Call `RMF_AudioCapture_Close()` to release resources, test open->start->stop->start->stop->close | current handle | RMF_SUCCESS | Should pass |
 */
 void test_l1_rmfAudioCapture_positive_RMF_AudioCapture_Close_complex (void)
```
Comment (hari22yuva) [inline src/test_l1_rmfAudioCapture.c:465]: we need to remove these Complex test case in L1 
please check other functions

File: src/test_l1_rmfAudioCapture.c [hunk 3]
```diff
@@ -1699,92 +1872,107 @@ void test_l1_rmfAudioCapture_negative_RMF_AudioCapture_Open_Type_mixed (void)
 */
 void test_l1_rmfAudioCapture_simultaneous_sessions (void)
 {
```
Comment (hari22yuva) [inline src/test_l1_rmfAudioCapture.c:1874]: these also need to be removed

**Review by hari22yuva — COMMENTED** [review]
Please check compilation for Linux , ARM it's should be zero warnings, errors

File: src/test_l1_rmfAudioCapture.c (hunk 2 above)
Comment (thanushreevani) [reply src/test_l1_rmfAudioCapture.c]: done

File: src/test_l1_rmfAudioCapture.c (hunk 3 above)
Comment (thanushreevani) [reply src/test_l1_rmfAudioCapture.c]: done

File: src/test_l1_rmfAudioCapture.c [hunk 4]
```diff
@@ -100,75 +100,75 @@ bool test_l1_is_aux_capture_supported()
 
 static bool test_l1_create_suite_of_positive_tests()
 {
-	const char * config = getenv("AC_GENERATE_POSITIVE_ONLY_SUITE");
-	if((NULL != config) && (0 == strncasecmp(config, "TRUE", 4)))
-		return true;
-	else
-		return false;
+    const char * config = getenv("AC_GENERATE_POSITIVE_ONLY_SUITE");
```
Comment (hari22yuva) [inline src/test_l1_rmfAudioCapture.c:103]: remove this one API

File: src/test_l1_rmfAudioCapture.c [hunk 5]
```diff
@@ -1799,37 +1420,31 @@ static UT_test_suite_t * pSuite = NULL;
  */
 int test_l1_rmfAudioCapture_register_positive_only_suite ( void )
```
Comment (hari22yuva) [inline src/test_l1_rmfAudioCapture.c:1421]: we can add easy way to add suite 
so remove this api also

**Review by hari22yuva — COMMENTED** [review]
please check this

File: src/test_l1_rmfAudioCapture.c (hunk 4 above)
Comment (thanushreevani) [reply src/test_l1_rmfAudioCapture.c]: done

File: src/test_l1_rmfAudioCapture.c (hunk 5 above)
Comment (thanushreevani) [reply src/test_l1_rmfAudioCapture.c]: done

**Review by hari22yuva — APPROVED** [review]
Approved

**Review by Sumip98 — APPROVED** [review]

**Pull request merged**